source/features/releases-tab.tsx

```diff
@@ -80,8 +80,13 @@ async function addReleasesTab(): Promise<false | void> {
 	// Trigger a reflow to push the right-most tab into the overflow dropdown (second attempt #4254)
 	window.dispatchEvent(new Event('resize'));
 
+	const dropdownMenu = await elementReady('.js-responsive-underlinenav .dropdown-menu ul', {waitForChildren: false})
+	if (!dropdownMenu) {
+		return
```

Comment:
We expect the dropdown to exist; if it doesn't, we want errors to tell us we should update the code.

You can use `dropdownMenu!` inside `appendBefore` to silence the typescript issue

Reply:
So I remove the guard and let the underlying `parent.append` trigger an error right?